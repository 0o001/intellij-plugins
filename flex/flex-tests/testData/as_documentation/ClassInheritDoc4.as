package {

/**
 * Subclass docs line1
 * Subclass docs line2
 * @inheritDoc
 * Subclass docs line3
 * Subclass docs line4
 *
 */
public interface ClassInheritDo<caret>c4 extends IBase {

}
}
