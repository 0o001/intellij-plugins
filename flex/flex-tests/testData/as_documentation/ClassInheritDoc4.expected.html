<div class='definition'><pre>ClassInheritDoc4</pre></div><div class='content'><p> Subclass docs line1<p>
 Subclass docs line2<p>
 Subclass docs line3<p>
 Subclass docs line4<p>
<p>
 Base docs line1<p>
 Base docs line2<p>
 Base docs line3<p>
 Base docs line4</div><table class='sections'></table>